whats_fresh/whats_fresh_api/forms.py

```diff
@@ -89,3 +90,50 @@ class Meta:
             'name': forms.TextInput(attrs={'required': 'true'}),
             'image': forms.FileInput(attrs={'required': 'true'})
         }
+
+
+class ThemeForm(forms.ModelForm):
+
+    class Meta:
+        model = Theme
+        exclude = []
+        widgets = {
+            'name': forms.TextInput(attrs={'required': 'true'}),
+            'background_color': forms.TextInput,
+            'foreground_color': forms.TextInput,
+            'logo': forms.FileInput,
+            'slogan': forms.TextInput,
+            'site_title': forms.TextInput(attrs={'required': 'true'}),
+            'vendors': forms.TextInput(attrs={'required': 'true'}),
+            'vendors_slug': forms.TextInput(attrs={'required': 'true'}),
+            'products': forms.TextInput(attrs={'required': 'true'}),
+            'vendors_slug': forms.TextInput(attrs={'required': 'true'}),
+            'vendors': forms.TextInput(attrs={'required': 'true'}),
+            'vendors_slug': forms.TextInput(attrs={'required': 'true'}),
+            'vendors': forms.TextInput(attrs={'required': 'true'}),
+            'vendors_slug': forms.TextInput(attrs={'required': 'true'}),
+            'vendors': forms.TextInput(attrs={'required': 'true'}),
+            'vendors_slug': forms.TextInput(attrs={'required': 'true'}),
+            'active': forms.TextInput
+
+        }
+
+
+class ThemeAdminForm(forms.ModelForm):
+
+    class Meta:
+        model = Theme
+        exclude = []
+
+    def clean(self):
```

Comment:
I'd like to have tests for this. Basically, test what happens if I give it a whole bunch of overlapping ones? No overlapping ones? We can manually test it but it's always nicer to have automated tests :) http://dashdrum.com/blog/2012/10/unit-testing-forms-in-django/